packages-proprietary/tokens/src/components/ams/card.tokens.json

```diff
@@ -2,11 +2,11 @@
   "ams": {
     "card": {
       "heading": {
-        "margin-block-end": { "value": "{ams.space.s}" }
+        "margin-block-end": { "value": "0" }
```

Comment:
This feels a little cramped, doesn't really fit with the spacious feel of our design imo

Reply:
The line height of our text does enough here. And this makes a shared rhythm of vertical whitespace between multiple lines of headings and paragraphs, which unifies them.

@Bondt007 and I did consider `space.xs` here but decided that `0` was better.

Reply:
> And this makes a shared rhythm of vertical whitespace between multiple lines of headings and paragraphs, which unifies them.

I assumed we would want to avoid that? Headings are a group which should be separated from paragraphs as a group, with a different space value than the lines within those groups. Gestalt and all.

Reply:
Unifiying and separating visual aspects can go hand in hand. The blue colour, the large font and the stronger font weight make it easy to recognize where the Card Heading ends and where body text begins, while said rhythm shows that they still belong together and helps reading them as one.

Reply:
As discussed with the entire team, we’ve restored the xs space below the Heading.